rules/linux/credential_access_bruteforce_passowrd_guessing.toml

```diff
@@ -16,6 +16,37 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Password Spraying"
+note = """## Triage and analysis
+
```

Comment:
Should we add `### Investigating Potential SSH Password Spraying` as a header here to remain consistent with the flow of other investigation guides?